web_widget_x2many_2d_matrix/static/src/js/abstract_view_matrix_limit_extend.js

```diff
@@ -0,0 +1,24 @@
+odoo.define('web_widget_x2many_2d_matrix.matrix_limit_extend', function (require) {
+    "use strict";
+
+var AbstractView = require('web.AbstractView');
+
+var AbstractViewCustom = AbstractView.include({
```

Comment:
should not be applied only to tree views?

Reply:
See https://github.com/odoo/odoo/blob/11.0/addons/web/static/src/js/views/abstract_view.js#L270
We are applying the same approach here.

Reply:
I know, I was just wondering if it wouldn't be better to apply the change to ListView only (which inherits from here: AbstractView -> BasicView -> ListView)

Reply:
I just tried changing to the ListView, but _setSubViewLimit is never being called, so it crashes.